conf/application.conf

```diff
@@ -118,12 +118,19 @@ microservice {
         port = 6795
         fetch-notification-uri = /cds-file-upload/notification
         fetch-declaration-status = /cds-file-upload/declaration-information
+        fetch-verified-email = /cds-file-upload/eori-email
       }
 
       contact-frontend {
         url = "http://localhost:9250/contact/beta-feedback-unauthenticated"
         service-id = "SFUS"
       }
+
+      customs-email-frontend {
+        protocol = http
+        host = localhost
+        port = 9898
+      }
```

Comment:
What is email-frontend used for ? Is that not a difference piece of work?

Reply:
This is the service we offer a link to the user to. They follow the link to verify their email address.